notebooks/official/experiments/get_started_with_comparing_local_trained_models.ipynb

```diff
@@ -0,0 +1,1027 @@
+{
```

Comment:
<p>Install the following packages required for execution of this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>done.</p>